src/express/index.js

```diff
@@ -12,11 +18,15 @@ const app = express();
 
 app.set(`views`, path.resolve(__dirname, ServerConfig.TEMPLATES));
 app.set(`view engine`, `pug`);
-
 app.use(express.static(path.resolve(__dirname, ServerConfig.PUBLIC_DIR)));
-
 app.use(`/`, mainRoutes);
 app.use(`/my`, myRoutes);
 app.use(`/offers`, offersRoutes);
 
-app.listen(ServerConfig.DEFAULT_PORT);
+app.listen(ServerConfig.DEFAULT_PORT, (err) => {
+  if (err) {
+    return showErrorMessage(`Ошибка при создании сервера: ${err.message}`);
+  }
+
+  return showSuccessMessage(`Ожидаю соединений на http://localhost:${ServerConfig.DEFAULT_PORT}`);
```

Comment:
```suggestion
  return showSuccessMessage(`Ожидаю соединений на ${ServerConfig.DEFAULT_PORT}`);
```
Это лишнее. Когда ты запускаешь на продакшен сервере, это не нужно.